modules/k8s-cluster/security.tf

```diff
@@ -109,6 +109,28 @@ resource "aws_security_group_rule" "nodes-from-vpc" {
   cidr_blocks = ["${data.aws_vpc.private.cidr_block}"]
 }
 
+resource "aws_security_group_rule" "nodes-from-controller" {
+  security_group_id = "${aws_security_group.node.id}"
+
+  type      = "ingress"
+  protocol  = "tcp"
+  from_port = 1025
+  to_port   = 65535
+
+  source_security_group_id = "${aws_security_group.controller.id}"
+}
+
+resource "aws_security_group_rule" "controller-to-nodes" {
+  security_group_id = "${aws_security_group.controller.id}"
+
+  type      = "egress"
+  protocol  = "tcp"
+  from_port = 1025
+  to_port   = 65535
+
+  source_security_group_id = "${aws_security_group.node.id}"
+}
+
```

Comment:
I deliberately didn't replicate these rules when introducing the new security group because `nodes-from-controller` is already covered by `nodes-from-vpc` and `controller-to-nodes` is covered by `controller-egress`.

Reply:
🤔 somehow the `controller-egress` rule doesn't seem to exist in the sandbox console but i definitely saw terraform create it.